### DISCUSSION TITLE
Please provide a Release Artifact on each release

### DISCUSSION THREAD

**Discussion by rkottmann** [OP-TED/eForms-SDK#38] [Ideas]
Currently (at least until 0.6.2), you do a GitHub release, but you do not provide a release artifact.

Please provide one, which is compiled version of the sources. At minimum you could exclude all the build specific stuff like `.mvn` `.github` etc.

**Comment by rousso** [comment]
Hi @rkottmann ,

We do publish an artifact for each release on [Maven Central](https://search.maven.org/search?q=g:eu.europa.ted.eforms). 
We will make sure to add this information in the release notes in the future, so that it is easier to spot for developers.

Thank you for pointing that out.

**Comment by bertrand-lorentz** [comment]
Just a small clarification: the eForms SDK does not currently contain any sources that can be compiled.
So the artifact on Maven Central has practically the same content as the source code ZIP attached to each release.
